app/views/posts/index.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Here are the posts for user @posts</h1>
+<div class="user_find_container">
+    <div class="user_img_div">
+      <img class="user_img" src="<%= @user.photo %>">
+    </div>
+    <div class="user_internal">
+      <h1><%= @user.name %></h1>
+      <p class="post_count">Number of Posts: <%= @user.posts_counter %></p>
+    </div>
+</div>
+
+<% @posts = Post.where(author_id: @user.id) %>
+<% if @posts.present? %>
+  <% @posts.each do |post| %>
+  <a class="each_post_detail" href="<%= user_post_path(@user, post) %>">
+  <div class="post_each_post">
+    <h3>Title: <%= post.title %></h3>
+    <p>Text: <%= post.text %></p>
+    <div class="post_like_comment_div">
+      <p>Comments: <%= post.comments_counter %>,
+      Likes: <%= post.likes_counter %></p>
+    </div>
+  </div>
+  </a>
+  <% @comments = Comment.where(post_id: post.id) %>
+  <div class="all_posts">
+  <% @comments.each do |comment| %>
```

Comment:
- [ ] [OPTIONAL] It would be more descriptive to use the `five_most_recent_comments` method here instead of adding it into the controller